src/components/carousel/carousel.svelte

```
@@ -0,0 +1,77 @@
+	function prevSlide() {
+		currentSlide = (currentSlide - 1 + images.length) % images.length;
+	}
+</script>
+
+<div class="carousel">
+	<div class="slides" style="transform: translateX(-{currentSlide * 100}%);">
+		{#each images as image, i}
+			<div class="slide">
+				<img src={image.src} alt={image.alt} />
+			</div>
+		{/each}
```

ESLint

[error] Each block should have a key

(svelte/require-each-key)